src/bindings/c/tests/CMakeLists.txt

```diff
@@ -6,13 +6,13 @@ set(TARGET_NAME "InferenceEngineCAPITests")
 
 # Find OpenCV components if exist
 find_package(OpenCV COMPONENTS core imgproc imgcodecs QUIET)
-if(NOT OpenCV_FOUND)
-    message(WARNING "OPENCV is disabled or not found, ${TARGET_NAME} is disabled")
-    return()
-endif()
 
 add_executable(${TARGET_NAME} ie_c_api_test.cpp test_model_repo.hpp)
 
+if(OpenCV_FOUND)
+    target_compile_definitions(${TARGET_NAME} PRIVATE BUILD_OPENCV_DEPENDENT_TESTS)
```

Comment:
looks like you've removed OpenCV usage at all, is not it?

Reply:
Thanks, I've cleaned up this file too.